src/components/legend/color.tsx

```diff
@@ -34,32 +33,34 @@ interface ColorProps {
  * @internal
  */
 export const Color = memo(
-  forwardRef<HTMLDivElement, ColorProps>(({ color, isSeriesHidden = false, hasColorPicker, onClick }, ref) => {
+  forwardRef<HTMLButtonElement, ColorProps>(({ color, isSeriesHidden = false, hasColorPicker, onClick }, ref) => {
     if (isSeriesHidden) {
       return (
-        <button type="button" className="echLegendItem__color" aria-label="series hidden" title="series hidden">
+        <div className="echLegendItem__color" title="series hidden">
           {/* changing the default viewBox for the eyeClosed icon to keep the same dimensions */}
-          <Icon type="eyeClosed" viewBox="-3 -3 22 22" />
-        </button>
+          <Icon type="eyeClosed" viewBox="-3 -3 22 22" aria-label="series color is hidden" />
```

Comment:
🤔 It's not the "series color" that's hidden, it's the series itself, right?

I think this might be more clear? (But I'm also no wordsmith so would love to hear from others)

```suggestion
          <Icon type="eyeClosed" viewBox="-3 -3 22 22" aria-label="series ${seriesName} is hidden" />
```

Also, how does elastic-charts handle internationalization? We need to make sure these strings are i18n-able for Kibana.

Reply:
Not at the moment, we have just a very limited set of strings at the moment, but one next step will be to add i18n support

Reply:
@rshen91 can you fix the wording of this as suggested by Michael?

Reply:
Yes thank you good catch - please see ecc7797